test/tuple.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+include("testhelpers/OffsetArrays.jl")
```

Comment:
Guard this:

```suggestion
isdefined(Main, :OffsetArrays) || @eval Main include("testhelpers/OffsetArrays.jl")
```

Reply:
Ah, I don't think we are in the module `Main` here, I believe we still need to do `using .Main.OffsetArray` afterwards.